apps/dashboard/app/apps/ood_app.rb

```diff
@@ -308,7 +308,13 @@ def passenger_railsdb_app?
 
   # @return [String] memoized version string
   def version
-    @version ||= (version_from_file || version_from_git || "unknown").strip
+    @version ||= if Configuration.Configuration.hide_app_version?
+      nil
+    elsif version_from_file.to_s.downcase == 'unknown'
+      nil
+    else
+      (version_from_git).strip
```

Comment:
It think we still want `version_from_file` here in the else block.

Also - can you format this block a little better, try to get the if/else block to the right of the `||=` operator.